installing_on_kubernetes/_topics/migrate_to_appliance.md

```diff
@@ -0,0 +1,102 @@
+## Migrating from Kubernetes to Appliances
+
+### Prerequisites
+1. Deploy an appliance with a database disk.  It is recommended that the appliance being deployed is the same code version as the kubernetes environment.
```

Comment:
Do we want to provide instructions to support going to a newer version?

I feel like it's safer to migrate to the same version and then do an immediate upgrade via rpm.  What do you think?

Reply:
For followup